SETUP_LOCAL_CLUSTER.md

```diff
@@ -0,0 +1,72 @@
+# Introduction
+
+The best way to learn and develop the HOPR protocol is by interacting with a HOPR node connected inside the HOPR network. A series of HOPR nodes interconnected (i.e. reachable and with channels opened against each other) is called a HOPR cluster.
+Without a HOPR cluster, app developers can only fake interactions with a HOPR node by mocking their API[^1], and
+core protocol developers can't replicate critical functionality over messaging such as ticket redemption
+and packet relaying.
+
+Although the HOPR Association provides a [production HOPR cluster](https://status.hoprnet.org/) anyone could talk to in production[^2],
+this setup is not ideal for development as doing so would incur on costs for executing basic operations. Staging networks
+such as görli are too slow to provide meaningful feedback, so the ideal setup for properly interacting with the HOPR
+ecosystem toolset is running a local HOPR cluster, either in your workstation or via a cloud environment like
+[Gitpod](https://gitpod.io). This way contributors are not constrained by internet connection or blockchain RPC providers.
+
+# Tooling
+
+As of the time of writing, the best way to set up a local HOPR cluster is by following these steps[^3]. Please bear in mind that these steps
+had been tested only on `macOS` and `Linux` devices, for Windows please use our cloud setup instead.
+
+1. **Download the latest version of the repository**: Download a local version of our [GitHub repository monorepo](https://github.com/hoprnet/hoprnet)[^4]
+   and unzip it in your local folder (roughly `~30 Mbs` at the time of writing). For the next tasks, we will assume you are within that folder.
+
+```
+wget https://github.com/hoprnet/hoprnet/archive/refs/heads/master.zip
```

Comment:
why not use git? as we are targeting developers

Reply:
Cloning with `git` takes way more space since it also downloads all the git history.